SPECS/cri-o/cri-o.spec

```diff
@@ -0,0 +1,1711 @@
+#
+# Copyright (c) 2018 SUSE LINUX GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+
+%define debug_package %{nil}
+%define project github.com/cri-o/cri-o
+#Compat macro for new _fillupdir macro introduced in Nov 2017
+%if ! %{defined _fillupdir}
+  %define _fillupdir %{_var}/adm/fillup-templates
+%endif
+Summary:        OCI-based implementation of Kubernetes Container Runtime Interface
+# Define macros for further referenced sources
+Name:           cri-o
+Version:        1.21.2
+Release:        2%{?dist}
+License:        Apache-2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/cri-o/cri-o
+#Source0:       https://github.com/%{name}/%{name}/archive/refs/tags/v%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+# Below is a manually created tarball, no download link.
+# We're using pre-populated Go modules from this tarball, since network is disabled during build time.
+# How to re-build this file:
+#   1. wget https://github.com/cri-o/cri-o/archive/refs/tags/v%%{version}.tar.gz -O %%{name}-%%{version}.tar.gz
+#   2. tar -xf %%{name}-%%{version}.tar.gz
+#   3. cd %%{name}-%%{version}
+#   4. go mod vendor
+#   5. tar  --sort=name \
+#           --mtime="2021-04-26 00:00Z" \
+#           --owner=0 --group=0 --numeric-owner \
+#           --pax-option=exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime \
+#           -cf %%{name}-%%{version}-vendor.tar.gz vendor
+#
+Source1:        %{name}-%{version}-vendor.tar.gz
+Source2:        crio.service
+Source3:        sysconfig.crio
+Source4:        crio.conf
+Source5:        cri-o-rpmlintrc
+Source6:        kubelet.env
+BuildRequires:  btrfs-progs-devel
+BuildRequires:  device-mapper-devel
+BuildRequires:  fdupes
+BuildRequires:  glib2-devel
+BuildRequires:  glibc-devel
+BuildRequires:  golang
+BuildRequires:  golang-packaging
+BuildRequires:  gpgme-devel
+BuildRequires:  libapparmor-devel
+BuildRequires:  libassuan-devel
+BuildRequires:  libseccomp-devel
+BuildRequires:  sed
+Requires:       cni
+Requires:       cni-plugins
+Requires:       conmon
+Requires:       conntrack-tools
+Requires:       iproute
+Requires:       iptables
+Requires:       libcontainers-common >= 0.0.1
+Requires:       moby-runc
+Suggests:       katacontainers
+# Provide generic cri-runtime dependency (needed by kubernetes)
+Provides:       cri-runtime
```

Comment:
Could you also add version and release for `Provides`: `Provides: cri-runtime = %{version}-%{release}`?